components/vocabularies/hgnc-remote/api/src/main/java/org/phenotips/vocabulary/internal/RemoteGeneNomenclature.java

```diff
@@ -75,6 +75,9 @@
 @Singleton
 public class RemoteGeneNomenclature implements Vocabulary, Initializable
 {
+    /** The list of supported categories for this vocabulary. */
+    private static final Collection<String> SUPPORTED_CATEGORIES = Collections.emptyList();
```

Comment:
This is the same as `GeneNomenclature.java`, except that results are fetched over the network instead of from a local Solr index. So, the category should be the same: `gene`.